src/openrct2/scripting/ScRide.hpp

```diff
@@ -131,6 +546,79 @@ namespace OpenRCT2::Scripting
             }
         }
 
+        int16_t buildDate_get() const
```

Comment:
Shouldn't we be getting age rather than build date as this value always needs to be combined with the current month to get a useful value

Reply:
At the same time, setting it would then need the reverse calculating in the plugin. Should I just add a new field for age?

Reply:
I think what you did is fine.

Reply:
I think what you did is fine.

Reply:
Okay I won't remove this. If @duncanspumpkin thinks there should also be age then I will add that as an additional field. It is possible to subtract elapsedmonths in the script though.

Reply:
Just add a note then that this is in terms of months and to use with elapsed months to get the age/time since last refurbish.

Reply:
Would it be possible to create a JS getter for the age in addition to this buildDate field?

Reply:
Yes, would you want a setter as well?